.circleci/prepare.sh

```diff
@@ -1,6 +1,6 @@
 $PYTHON --version
 $PYTHON -m pip --version
-$PYTHON -m pip install -q --user --ignore-installed --upgrade virtualenv
+$PYTHON -m pip install -q --user --ignore-installed --upgrade virtualenv<20
```

Comment:
why here and in macos.py?

Reply:
Because currently `virtualenv==20` seems to break the `pip` installation, as explained in the issue